package.json

```diff
@@ -58,34 +58,34 @@
     "js-yaml": "^3.14.0",
     "markdown-table": "^2.0.0",
     "prompt": "^1.0.0",
-    "serverless": "^1.75.1"
+    "serverless": "^2.7.0"
```

Comment:
out of curiosity, is it auto generated number?

Reply:
Just the newest version of the package provided by `npx npm-check-updates -u`